resources/benchmark.ts

```diff
@@ -376,9 +376,12 @@ interface BenchmarkSample {
 function sampleModule(modulePath: string): BenchmarkSample {
   const sampleCode = `
     import fs from 'node:fs';
+    import os from 'node:os';
 
     import { benchmark } from '${modulePath}';
 
+    os.setPriority(os.constants.priority.PRIORITY_HIGHEST);
+
```

Comment:
@yaacovCR Also, you can try setting priority for benchmark itself to the lowest.
That will force context switches to occur before/after measurments.

Reply:
Or maybe benchmark will never finish? I think we can see what happens with this as a first step and take it from there.